src/bundle/Editor.ts

```diff
@@ -123,6 +123,8 @@ export type MarkupConfig = {
      * See more https://codemirror.net/docs/ref/#state.EditorState.languageDataAt
      */
     languageData?: YfmLangOptions['languageData'];
+    /** Autocompletion config. Can be used to style tooltip */
```

Comment:
Let's replace the comment with the fact that this is the prosemirror autocomplete config